src/ui/connectionmanager.cc

```diff
@@ -318,21 +319,27 @@ void ConnectionNotificationWidget::updateConnectionStatus(
     last_status_change_ = frame_;
   }
 
+  QString status_text;
   switch (connection_status) {
   case client::NetworkClient::ConnectionStatus::NotConnected:
-    ui_->connection_status_text_label->setText("Not connected");
+    status_text = "Not connected";
```

Comment:
Hmm, I don't know if displaying `Not connected:<currently selected profile>` is what I'd expect.